src/Adjudications/AdjudicationHistory/AdjudicationHistoryFormValidation.js

```diff
@@ -0,0 +1,21 @@
+import { FORM_ERROR } from 'final-form'
+
+const validateThenSubmit = ({ fromDate, toDate, submit }) => values => {
+  if (toDate && fromDate && fromDate.isAfter(toDate)) {
+    return {
+      [FORM_ERROR]: [
+        {
+          targetName: 'fromDate',
+          text: 'The From date must be the same or before the To date',
```

Comment:
missing `as` i.e. should be:
```The From date must be the same as or before the To date```
Same with `toDate`

Reply:
fixed this